src/mesh_sandbox/routers/admin.py

```diff
@@ -123,6 +123,12 @@ async def add_message_event(
     status_code=status.HTTP_200_OK,
     response_model_exclude_none=True,
 )
+@router.get(
+    "/messageexchange/mailbox/{mailbox_id}",
+    summary=f"Get mailbox details. {TESTING_ONLY}",
```

Comment:
Remove the TESTING_ONLY message for this item

Reply:
i've removed it.